robocop/checkers/comments.py

```diff
@@ -50,6 +50,93 @@ def visit_KeywordCall(self, node):  # noqa
         self.find_comments(node)
         self.generic_visit(node)
 
+    def visit_Return(self, node):  # noqa
```

Comment:
I've checked your code and I've noticed that we actually don't need visit_Keyword and visit_TestCase visitors. I've removed them and it worked fine. Can you also verify it?

Reply:
I needed to add it back because it couldn't discover one of the cases. Now it works fine.